Cargo.toml

```diff
@@ -9,7 +9,7 @@ name = "matrix-sdk-crypto-nodejs"
 readme = "README.md"
 repository = "https://github.com/matrix-org/matrix-rust-sdk-nodejs"
 rust-version = "1.70"
-version = "0.6.0"
+version = "0.1.0-beta.9"
```

Comment:
Let's make this `0.0.0` since the crate never gets published anyways. The WASM bindings do the same: https://github.com/matrix-org/matrix-rust-sdk-crypto-wasm/blob/4b8a1f63b40bd4ace045dc83ded03505591eeb7c/Cargo.toml#L12